src/gui/PluginBrowser.cpp

```diff
@@ -171,8 +192,9 @@ void PluginDescWidget::mousePressEvent( QMouseEvent * _me )
 {
 	if( _me->button() == Qt::LeftButton )
 	{
-		new StringPairDrag( "instrument", m_pluginDescriptor.name,
```

Comment:
mmm memory leak? (it has nothing to do with this PR, I just haven't seen anything like it)

Reply:
[Probably not](https://doc.qt.io/archives/qt-4.8/dnd.html): "The QMimeData and QDrag objects created by the source widget should not be deleted - they will be destroyed by Qt".

This makes sense: `StringPairDrag` passes `this` to its parent class `QDrag` which passes it to Qt. `QDrag` is not being deleted in this drag yet, but only after the drop (by Qt).